README.md

```diff
@@ -165,6 +165,12 @@ The value of the `theme-for` attribute can be a space-separated list of element
 
 When creating a theme module for an element, the styles in that theme module will apply to all instances of that element. The styles are always “global” in that sense and can’t be scoped by default with anything.
 
+### Theme module import order
+
+`ThemableMixin` has a theme overriding mechanism. To import the style modules in correct order they should be imported in the following order:
```

Comment:
This isn't stated correctly. The import order of the modules doesn't matter! It's all about the naming of the module IDs.

The point is that we should emphasize app developers not to prefix their style module IDs with `vaadin-`, `lumo-` or `-material` unless they specifically plan the modules to be overridden by Vaadin modules / other custom modules. Only use-case I can come up for this would be if someone's building a new base theme for the components, and this is such a rare use-case that it doesn't even need to be mentioned in the documentation.